examples/ui-demo/src/app/layout.tsx

```diff
@@ -23,10 +28,26 @@ export default function RootLayout({
     headers().get("cookie") ?? undefined
   );
 
+  const initialConfig = cookieToInitialConfig(headers().get("cookie"));
+
+  const classes = initialConfig
+    ? generateClassesForRoot(initialConfig)
+    : ["light"];
+  const styles = initialConfig ? generateStylesForRoot(initialConfig) : [];
+
   return (
-    <html lang="en" className="light">
+    <html
+      lang="en"
+      className={classes.join(" ")}
+      style={styles.reduce((accum, [key, value]) => {
+        accum[key] = value;
+        return accum;
+      }, {} as Record<string, string>)}
```

Comment:
Is this expression with reduce the same thing as
```ts
style={Object.fromEntries(styles)}
```
?

Reply:
I did not know about Object.fromEntries, neat!